diff --git a/sherpa-onnx/csrc/speaker-embedding-extractor-nemo-impl.h b/sherpa-onnx/csrc/speaker-embedding-extractor-nemo-impl.h
--- a/sherpa-onnx/csrc/speaker-embedding-extractor-nemo-impl.h
+++ b/sherpa-onnx/csrc/speaker-embedding-extractor-nemo-impl.h
@@ -130,9 +130,9 @@ class SpeakerEmbeddingExtractorNeMoImpl {

     auto EX = m.colwise().mean();
     auto EX2 = m.array().pow(2).colwise().sum() / num_frames;
-    auto variance = (EX2 - EX.array().pow(2)).max(1e-5);
+    auto variance = (EX2 - EX.array().pow(2)).max(1e-5f);

     auto stddev = variance.array().sqrt();

-    m = (m.rowwise() - EX).array().rowwise() / (stddev.array() + 1e-5);
+    m = (m.rowwise() - EX).array().rowwise() / (stddev.array() + 1e-5f);
   }
